static/app/views/detectors/detail.tsx

```diff
@@ -40,3 +96,16 @@ function Actions() {
     </Fragment>
   );
 }
+
+const PrioritiesList = styled('div')`
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: repeat(${priorities.length}, 1fr);
+  gap: ${space(0.5)};
+  width: auto;
```

Comment:
Couldn't figure out how to make the width more narrow :/ Looks silly right now

Reply:
`flex-grow: 0` or adjusting the `grid-template-columns` to something like `minmax(0, max-content) 1rem minmax(0, max-content)`?